packages/website/src/pages/index.tsx

```diff
@@ -108,8 +109,12 @@ const features: FeatureItem[] = [
 function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
     <div className="col col--12 padding-vert--lg">
-      <h2 className="text--center">{title}</h2>
-      {description}
+      <div className="text--center">
+        <Heading as="h2" id="start-typescript-eslint-title">
```

Comment:
@kmin-jeong this isn't right either. Let me try to explain from the start.

This `Feature` function component is rendered multiple times on the page. Each place it's rendered will need a unique ID.

The point of the ID is that someone can use it to link to that particular heading in the page. For example, `https://typescript-eslint.io/docs/#details` links to the heading in https://typescript-eslint.io/docs that has `id="details"`.

Right now it's hardcoded to have the same ID each time: `"start-typescript-eslint-title"`. Which is not good: if two headings have the same ID, linking to the URL for that ID could mean either of them. This is what I meant by "ambiguous" in https://github.com/typescript-eslint/typescript-eslint/pull/5951#discussion_r1020767451.

All that being said, I tried this out locally and as it turns out, the `Heading` component automatically adds a good `id` by default! So we don't need to provide one at all.

```suggestion
        <Heading as="h2">
```

That should simplify things. 😄

Reply:
> automatically adds a good `id` by default!

I'd be surprised—I don't think we've added that logic 😕 If there's no `id`, it should render a heading with no `#` link at all.

Reply:
Huh, you're right! Sorry about that! I must have been looking at a cached version or something. I'll edit my post.

Reply:
Yeah, sometimes the hot reload lags behind. Always a good idea to refresh.

Reply:
@JoshuaKGoldberg  ok! I modified `"start-typescript-eslint-title"` to `id = {title}`.
I used regex in replace() and toLowerCase(). It replace the space or `( , )` or `( _ )` with a `"-"` .

Reply:
+ I updated the branch and pulled it to the feat/5949 branch, but the following error occurs.I'm looking for an issue, but I don't see a clear solution. I use window10 and visual studio code.
![errors](https://user-images.githubusercontent.com/53456037/202166612-ecb60292-ed80-4f14-9a0d-47c0d47e5ca5.jpg)

Reply:
Oh yeah sorry that was me, unrelated. Fixed in #6001.

Reply:
after updated branch,It's resolved!